ui/src/components/Map/LeafletMap.vue

```diff
@@ -233,10 +279,16 @@ const tileProviders = [
   },
 ]
 
+onMounted(async () => {
+  store.dispatch('mapModule/getNodes')
```

Comment:
I know this is a pattern everywhere for how we're implementing vuex so this is really a tech-debt/future comment, but these bare strings should really be captured in constants and kept close to their implementation.

We should also look at replacing vuex with pinia to cut down on the store boilerplate as well (and because pinia is the official store now for vue3)

Reply:
I agree. I will open a tech debt ticket for moving to pinia as well as replacing store dispatch action strings with constants.